### ISSUE TITLE
Forge mods are not properly filtered out from Modrinth results

### ISSUE THREAD

**Issue by John-Paul-R** [John-Paul-R/fibermc#34]
As part of fixing this, bad values should be filtered out of fiberdb. (Alternatively, apply filtering internally)

**Comment by John-Paul-R** [comment]
resolved this in [FibermcCsr/a8fcd34236163396e2be969ba4715958e1c5c5f2](https://github.com/John-Paul-R/FibermcCsr/commit/a8fcd34236163396e2be969ba4715958e1c5c5f2)

**Comment by VaelophisNyx** [comment]
This appears to still be happening

**Comment by John-Paul-R** [comment]
You're right, I found another bug that was causing them to be re added. Need to clean up what I have stored again. Ty for bringing to my attention.
